.github/workflows/testing.yml

```diff
@@ -40,5 +46,9 @@ jobs:
       - name: "Install dependencies"
         run: python -m pip install tox tox-gh-actions
 
-      - name: "Run tox targets for ${{ matrix.python-version }}"
+      - name: "Run tox targets on Ubuntu for ${{ matrix.python-version }}"
+        if: matrix.os == 'ubuntu-latest'
+        run: "python -m tox"
+      - name: "Run tox targets on MacOS for ${{ matrix.python-version }}"
+        if: matrix.os == 'macos-latest'
         run: "python -m tox"
```

Comment:
You don't need to separate these since they are running the same command :slightly_smiling_face: The reason why it's different in ERT is that they have slightly different commands!

Reply:
Fair point